src/neo/Persistence/IReadOnlyStore.cs

```diff
@@ -8,8 +8,8 @@ namespace Neo.Persistence
     /// </summary>
     public interface IReadOnlyStore
     {
-        IEnumerable<(byte[] Key, byte[] Value)> Seek(byte table, byte[] key, SeekDirection direction);
-        byte[] TryGet(byte table, byte[] key);
-        bool Contains(byte table, byte[] key);
+        IEnumerable<(byte[] Key, byte[] Value)> Seek(byte[] key, SeekDirection direction);
```

Comment:
Could we change byte table, to 'contract Id', in order to make it easier the MPT exclusion ?

Reply:
The first four bytes is the contract id. I think it's easy for the plugin to exclude it.